test/unit/main_test.ts

```diff
@@ -80,6 +80,8 @@ describe('Visitor pattern', () => {
   it('renames identifiers of property declarations/assignments', () => {
     expectTranslate('var foo = { bar: { baz: 12; } }; foo.bar.baz;')
       .to.equal('var foo = { $: { _: 12; } }; foo.$._;');
+    expectTranslate('var x = "something"; var foo = { bar: { baz: x; } }; foo.bar.baz;')
```

Comment:
This tests the case for renaming when we have identifiers nodes on both sides of the `:`. In the previous change, the `x` would have also gotten renamed in the `PropertyAssignment`.